eva_submission/xlsx/xlsx_validation.py

```diff
@@ -113,11 +113,11 @@ def group_of_fields_required(self, sheet_name, row, *args):
 
     def same_set(self, list1, list2, list1_desc, list2_desc):
         if not set(list1) == set(list2):
-            list1_list2 = set(list1).difference(list2)
-            list2_list1 = set(list2).difference(list1)
+            list1_list2 = sorted([str(e) for e in set(list1).difference(list2)])
+            list2_list1 = sorted([str(e) for e in set(list2).difference(list1)])
```

Comment:
Is this just for tests, or is it useful to have these sorted anyway?

Reply:
I think it can be nice to have the list of field sorted alphabetically but you're right I made that change toy accommodate the tests